style.css

```diff
@@ -12,12 +12,12 @@
     position: fixed;
     bottom: 0;
     right: 0;
-    right: calc(50vw - 525px);
+    right: calc(50vw - 550px);
     width: 300px;
     z-index: 1;
   }
   #rfc\.toc {
-    top: 15px;
+    top: 55px;
```

Comment:
This appears to push the TOC title into the TOC itself.  It's more than 25px high.

I think that the problem here is that the xml2rfc version puts this class on the header, whereas Julian's XSLT puts it on a \<nav> container around both the title and the content.  That's annoying because it means that there will need to be a new rule to override the rules that are used for xml2rfc:

```css
#rfc\.toc { top: 55px; overflow: auto; overscroll-behavior: contain; }
#rfc\.toc ul.toc { /* override all the other rules we just added here. */ position: unset; }
ul.toc { top: 80px; overflow: auto; overscroll-behavior: contain; }
```

This is why Julian doesn't like xml2rfc.  And I totally agree with him there.